druid/examples/multiwin.rs

```diff
@@ -66,14 +68,10 @@ fn ui_builder() -> impl Widget<State> {
     let text = LocalizedString::new("hello-counter")
         .with_arg("count", |data: &State, _env| data.menu_count.into());
     let label = Label::new(text);
-    let inc_button = Button::<State>::new("Add menu item").on_click(|ctx, data, _env| {
-        data.menu_count += 1;
-        ctx.set_menu(make_menu::<State>(data));
-    });
-    let dec_button = Button::<State>::new("Remove menu item").on_click(|ctx, data, _env| {
-        data.menu_count = data.menu_count.saturating_sub(1);
-        ctx.set_menu(make_menu::<State>(data));
-    });
+    let inc_button = Button::<State>::new("Add menu item")
+        .on_click(|ctx, data, _env| ctx.submit_command(MENU_INCREMENT_ACTION, ctx.window_id()));
```

Comment:
Do you need the target here? It looks to me like you're handling all the commands globally in the AppDelegate and ignoring the target anyway...

Reply:
The target should probably be `Global` to match the intent.